### PR TITLE
Databases week1

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

File: databases/week1/index.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(id) FROM task LIMIT 100;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT COUNT(id) FROM task WHERE `due_date` IS NULL LIMIT 100;
+Find all the tasks that are marked as done
+Find all the tasks that are not marked as done
+Get all the tasks, sorted with the most recently created first
+Get the single most recently created task
+Get the title and due date of all tasks where the title or description contains database
```
Comment (27ManishaPatel) [inline databases/week1/index.sql]: Question you can write as a comment 😊

File: databases/week1/index.sql [hunk 2]
```diff
@@ -2,11 +2,19 @@
...
-Get the single most recently created task
-Get the title and due date of all tasks where the title or description contains database
-Get the title and status (as text) of all tasks
+/* Find all the tasks that are marked as done */
+SELECT * FROM task JOIN status ON `status.id` = 'task.status_id' WHERE status.name = 'Done'
+
+/*Find all the tasks that are not marked as done*/
+
+SELECT * FROM task JOIN status ON status.id = task.status_id WHERE status.name != 'Done'
+/*Get all the tasks, sorted with the most recently created first*/
```
Comment (27ManishaPatel) [inline databases/week1/index.sql:11]: You can use  ' --Get all the tasks, sorted with the most recently created first ' for single line comments

File: databases/week1/index.sql [hunk 3]
```diff
@@ -15,6 +15,10 @@ SELECT * FROM task order by created desc limit 1
...
 /*Get the title and status (as text) of all tasks*/
+SELECT task.title, status.name FROM task JOIN status ON status.id = task.status_id
+/*Get the name of each status, along with a count of how many tasks have that status*/
+SELECT status.name, COUNT(*) as number FROM task JOIN status ON status.id = task.status_id GROUP BY status.name
 
-Get the name of each status, along with a count of how many tasks have that status
-Get the names of all statuses, sorted by the status with most tasks first*/
+/*Get the names of all statuses, sorted by the status with most tasks first*/
+
+SELECT status.name, COUNT(*) as number FROM task JOIN status ON status.id = task.status_id GROUP BY status.name ORDER BY number DESC
```
Comment (27ManishaPatel) [inline databases/week1/index.sql:24]: good work!

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls implement given feedback from Manisha.

**Pull request merged**